### PR TITLE
add gaap import and new optional argument

### PR DESCRIPTION
**Pull request by Sherlock-Holo**
add new optional argument poly_client_certificate_ids for
- tencentcloud_gaap_http_domain
- tencentcloud_gaap_layer7_listener

add new output argument poly_client_certificate_ids for
- tencentcloud_gaap_http_domains
- tencentcloud_gaap_layer7_listeners

deprecate optional argument client_certificate_id for
- tencentcloud_gaap_http_domain
- tencentcloud_gaap_layer7_listener

deprecate output argument client_certificate_id for
- tencentcloud_gaap_http_domains
- tencentcloud_gaap_layer7_listeners

### CODE REVIEW

**Comment by Sherlock-Holo** [comment]
# unit test log

## data source

> === RUN   TestAccDataSourceTencentCloudGaapCertificates_basic                                                          --- PASS: TestAccDataSourceTencentCloudGaapCertificates_basic (1.26s)                                                  === RUN   TestAccDataSourceTencentCloudGaapCertificates_name                                                           --- PASS: TestAccDataSourceTencentCloudGaapCertificates_name (0.94s)                                                   === RUN   TestAccDataSourceTencentCloudGaapCertificates_type                                                           --- PASS: TestAccDataSourceTencentCloudGaapCertificates_type (0.75s)                                                   === RUN   TestAccDataSourceTencentCloudGaapHttpDomains_basic                                                           --- PASS: TestAccDataSourceTencentCloudGaapHttpDomains_basic (34.50s)                                                  === RUN   TestAccDataSourceTencentCloudGaapHttpRules_basic                                                             --- PASS: TestAccDataSourceTencentCloudGaapHttpRules_basic (36.19s)                                                    === RUN   TestAccDataSourceTencentCloudGaapHttpRules_path                                                              --- PASS: TestAccDataSourceTencentCloudGaapHttpRules_path (32.94s)                                                     === RUN   TestAccDataSourceTencentCloudGaapHttpRules_forwardHost                                                       --- PASS: TestAccDataSourceTencentCloudGaapHttpRules_forwardHost (36.06s)                                              === RUN   TestAccDataSourceTencentCloudGaapLayer4Listeners_basic                                                       --- PASS: TestAccDataSourceTencentCloudGaapLayer4Listeners_basic (29.36s)                                              === RUN   TestAccDataSourceTencentCloudGaapLayer4Listeners_tcp                                                         --- PASS: TestAccDataSourceTencentCloudGaapLayer4Listeners_tcp (65.36s)                                                === RUN   TestAccDataSourceTencentCloudGaapLayer4Listeners_UDP                                                         --- PASS: TestAccDataSourceTencentCloudGaapLayer4Listeners_UDP (30.75s)                                                === RUN   TestAccDataSourceTencentCloudGaapLayer7Listeners_basic                                                       --- PASS: TestAccDataSourceTencentCloudGaapLayer7Listeners_basic (26.20s)                                              === RUN   TestAccDataSourceTencentCloudGaapLayer7Listeners_https                                                       --- PASS: TestAccDataSourceTencentCloudGaapLayer7Listeners_https (27.05s)                                              === RUN   TestAccDataSourceTencentCloudGaapProxies_basic                                                               --- PASS: TestAccDataSourceTencentCloudGaapProxies_basic (39.53s)                                                      === RUN   TestAccDataSourceTencentCloudGaapProxies_filter                                                              --- PASS: TestAccDataSourceTencentCloudGaapProxies_filter (41.36s)                                                     === RUN   TestAccDataSourceTencentCloudGaapRealservers_basic                                                           --- PASS: TestAccDataSourceTencentCloudGaapRealservers_basic (1.49s)                                                   === RUN   TestAccDataSourceTencentCloudGaapRealservers_domain                                                          --- PASS: TestAccDataSourceTencentCloudGaapRealservers_domain (1.28s)                                                  === RUN   TestAccDataSourceTencentCloudGaapRealservers_name                                                            --- PASS: TestAccDataSourceTencentCloudGaapRealservers_name (1.30s)                                                    === RUN   TestAccDataSourceTencentCloudGaapSecurityPolices_basic                                                       --- PASS: TestAccDataSourceTencentCloudGaapSecurityPolices_basic (8.23s)                                               === RUN   TestAccDataSourceTencentCloudGaapSecurityRules_basic                                                         --- PASS: TestAccDataSourceTencentCloudGaapSecurityRules_basic (11.63s)                                                === RUN   TestAccDataSourceTencentCloudGaapSecurityRules_multi                                                         --- PASS: TestAccDataSourceTencentCloudGaapSecurityRules_multi (14.91s)                                                PASS                                                                                                                   ok      github.com/terraform-providers/terraform-provider-tencentcloud/tencentcloud     441.108s

## resource

> === RUN   TestAccTencentCloudGaapCertificate_basic                                                                     --- PASS: TestAccTencentCloudGaapCertificate_basic (1.08s)                                                             === RUN   TestAccTencentCloudGaapCertificate_clientCA                                                                  --- PASS: TestAccTencentCloudGaapCertificate_clientCA (0.68s)                                                          === RUN   TestAccTencentCloudGaapCertificate_ServerSSL                                                                 --- PASS: TestAccTencentCloudGaapCertificate_ServerSSL (0.66s)                                                         === RUN   TestAccTencentCloudGaapCertificate_realserverCA                                                              --- PASS: TestAccTencentCloudGaapCertificate_realserverCA (0.65s)                                                      === RUN   TestAccTencentCloudGaapCertificate_ProxySSL                                                                  --- PASS: TestAccTencentCloudGaapCertificate_ProxySSL (0.66s)                                                          === RUN   TestAccTencentCloudGaapCertificate_updateName                                                                --- PASS: TestAccTencentCloudGaapCertificate_updateName (1.00s)                                                        === RUN   TestAccTencentCloudGaapHttpDomain_basic                                                                      --- PASS: TestAccTencentCloudGaapHttpDomain_basic (26.44s)                                                             === RUN   TestAccTencentCloudGaapHttpDomain_https                                                                      --- PASS: TestAccTencentCloudGaapHttpDomain_https (35.13s)                                                             === RUN   TestAccTencentCloudGaapHttpDomain_httpsMutualAuthentication                                                  --- PASS: TestAccTencentCloudGaapHttpDomain_httpsMutualAuthentication (28.84s)                                         === RUN   TestAccTencentCloudGaapHttpDomain_httpsPolyClientCertificateIds                                              --- PASS: TestAccTencentCloudGaapHttpDomain_httpsPolyClientCertificateIds (28.69s)                                     === RUN   TestAccTencentCloudGaapHttpDomain_httpsCCIdToPolyIds                                                         --- PASS: TestAccTencentCloudGaapHttpDomain_httpsCCIdToPolyIds (28.06s)                                                === RUN   TestAccTencentCloudGaapHttpRule_basic                                                                        --- PASS: TestAccTencentCloudGaapHttpRule_basic (36.19s)                                                               === RUN   TestAccTencentCloudGaapHttpRule_httpUpdate                                                                   --- PASS: TestAccTencentCloudGaapHttpRule_httpUpdate (45.57s)                                                          === RUN   TestAccTencentCloudGaapHttpRule_httpUpdateRealservers                                                        --- PASS: TestAccTencentCloudGaapHttpRule_httpUpdateRealservers (41.90s)                                               === RUN   TestAccTencentCloudGaapHttpRule_noHealth                                                                     --- PASS: TestAccTencentCloudGaapHttpRule_noHealth (37.37s)                                                            === RUN   TestAccTencentCloudGaapHttpRule_domainRealserver                                                             --- PASS: TestAccTencentCloudGaapHttpRule_domainRealserver (34.07s)                                                    === RUN   TestAccTencentCloudGaapLayer4Listener_basic                                                                  --- PASS: TestAccTencentCloudGaapLayer4Listener_basic (28.23s)                                                         === RUN   TestAccTencentCloudGaapLayer4Listener_TcpDomain                                                              --- PASS: TestAccTencentCloudGaapLayer4Listener_TcpDomain (27.88s)                                                     === RUN   TestAccTencentCloudGaapLayer4Listener_update                                                                 --- PASS: TestAccTencentCloudGaapLayer4Listener_update (36.09s)                                                        === RUN   TestAccTencentCloudGaapLayer4Listener_udp                                                                    --- PASS: TestAccTencentCloudGaapLayer4Listener_udp (29.35s)                                                           === RUN   TestAccTencentCloudGaapLayer4Listener_udpDomain                                                              --- PASS: TestAccTencentCloudGaapLayer4Listener_udpDomain (27.83s)                                                     === RUN   TestAccTencentCloudGaapLayer4Listener_udpUpdate                                                              --- PASS: TestAccTencentCloudGaapLayer4Listener_udpUpdate (30.54s)                                                     === RUN   TestAccTencentCloudGaapLayer7Listener_basic                                                                  --- PASS: TestAccTencentCloudGaapLayer7Listener_basic (26.09s)                                                         === RUN   TestAccTencentCloudGaapLayer7Listener_https                                                                  --- PASS: TestAccTencentCloudGaapLayer7Listener_https (27.65s)                                                         === RUN   TestAccTencentCloudGaapLayer7Listener_httpsTwoWayAuthentication                                              --- PASS: TestAccTencentCloudGaapLayer7Listener_httpsTwoWayAuthentication (25.47s)                                     === RUN   TestAccTencentCloudGaapLayer7Listener_httpsForwardHttps                                                      --- PASS: TestAccTencentCloudGaapLayer7Listener_httpsForwardHttps (25.48s)                                             === RUN   TestAccTencentCloudGaapLayer7Listener_httpsPolyClientCertificateIds                                          --- PASS: TestAccTencentCloudGaapLayer7Listener_httpsPolyClientCertificateIds (27.66s)                                 === RUN   TestAccTencentCloudGaapLayer7Listener_httpsCcToPoly                                                          --- PASS: TestAccTencentCloudGaapLayer7Listener_httpsCcToPoly (26.19s)                                                 === RUN   TestAccTencentCloudGaapProxy_basic                                                                           --- PASS: TestAccTencentCloudGaapProxy_basic (29.77s)                                                                  === RUN   TestAccTencentCloudGaapProxy_update                                                                          --- PASS: TestAccTencentCloudGaapProxy_update (58.13s)                                                                 === RUN   TestAccTencentCloudGaapRealserver_basic                                                                      --- PASS: TestAccTencentCloudGaapRealserver_basic (2.35s)                                                              === RUN   TestAccTencentCloudGaapRealserver_domain                                                                     --- PASS: TestAccTencentCloudGaapRealserver_domain (1.16s)                                                             === RUN   TestAccTencentCloudGaapRealserver_updateName                                                                 --- PASS: TestAccTencentCloudGaapRealserver_updateName (1.78s)                                                         === RUN   TestAccTencentCloudGaapRealserver_updateTags                                                                 --- PASS: TestAccTencentCloudGaapRealserver_updateTags (1.85s)                                                         === RUN   TestAccTencentCloudGaapSecurityPolicy_basic                                                                  --- PASS: TestAccTencentCloudGaapSecurityPolicy_basic (8.00s)                                                          === RUN   TestAccTencentCloudGaapSecurityPolicy_disable                                                                --- PASS: TestAccTencentCloudGaapSecurityPolicy_disable (8.28s)                                                        === RUN   TestAccTencentCloudGaapSecurityPolicy_drop                                                                   --- PASS: TestAccTencentCloudGaapSecurityPolicy_drop (7.85s)                                                           === RUN   TestAccTencentCloudGaapSecurityRule_basic                                                                    --- PASS: TestAccTencentCloudGaapSecurityRule_basic (10.62s)                                                           === RUN   TestAccTencentCloudGaapSecurityRule_drop                                                                     --- PASS: TestAccTencentCloudGaapSecurityRule_drop (10.94s)                                                            === RUN   TestAccTencentCloudGaapSecurityRule_name                                                                     --- PASS: TestAccTencentCloudGaapSecurityRule_name (12.28s)                                                            === RUN   TestAccTencentCloudGaapSecurityRule_ipSubnet                                                                 --- PASS: TestAccTencentCloudGaapSecurityRule_ipSubnet (11.03s)                                                        === RUN   TestAccTencentCloudGaapSecurityRule_allProtocols                                                             --- PASS: TestAccTencentCloudGaapSecurityRule_allProtocols (11.31s)                                                    === RUN   TestAccTencentCloudGaapSecurityRule_AllPorts                                                                 --- PASS: TestAccTencentCloudGaapSecurityRule_AllPorts (10.40s)                                                        PASS                                                                                                                   ok      github.com/terraform-providers/terraform-provider-tencentcloud/tencentcloud     871.224s

File: tencentcloud/extension_gaap.go [hunk 1]
```diff
@@ -1,9 +1,9 @@
 package tencentcloud
 
-const GAAP_PROXY_ID = "link-9ck3lzq7"
-
-// all gaap proxy status https://cloud.tencent.com/document/api/608/37023#ProxyInfo
 const (
+	GAAP_PROXY_ID = "link-9ck3lzq7"
```
Comment (likexian) [inline tencentcloud/extension_gaap.go]: 测试用的写到 basic_test.go

**Comment by Sherlock-Holo** [comment]
# unit test log

## data source

> === RUN   TestAccDataSourceTencentCloudGaapCertificates_basic                                                          --- PASS: TestAccDataSourceTencentCloudGaapCertificates_basic (1.35s)                                                  === RUN   TestAccDataSourceTencentCloudGaapCertificates_name                                                           --- PASS: TestAccDataSourceTencentCloudGaapCertificates_name (0.80s)                                                   === RUN   TestAccDataSourceTencentCloudGaapCertificates_type                                                           --- PASS: TestAccDataSourceTencentCloudGaapCertificates_type (0.77s)                                                   === RUN   TestAccDataSourceTencentCloudGaapHttpDomains_basic                                                           --- PASS: TestAccDataSourceTencentCloudGaapHttpDomains_basic (26.59s)                                                  === RUN   TestAccDataSourceTencentCloudGaapHttpRules_basic                                                             --- PASS: TestAccDataSourceTencentCloudGaapHttpRules_basic (34.07s)                                                    === RUN   TestAccDataSourceTencentCloudGaapHttpRules_path                                                              --- PASS: TestAccDataSourceTencentCloudGaapHttpRules_path (42.14s)                                                     === RUN   TestAccDataSourceTencentCloudGaapHttpRules_forwardHost                                                       --- PASS: TestAccDataSourceTencentCloudGaapHttpRules_forwardHost (33.96s)                                              === RUN   TestAccDataSourceTencentCloudGaapLayer4Listeners_basic                                                       --- PASS: TestAccDataSourceTencentCloudGaapLayer4Listeners_basic (29.61s)                                              === RUN   TestAccDataSourceTencentCloudGaapLayer4Listeners_tcp                                                         --- PASS: TestAccDataSourceTencentCloudGaapLayer4Listeners_tcp (63.74s)                                                === RUN   TestAccDataSourceTencentCloudGaapLayer4Listeners_tcp                                                         === RUN   TestAccDataSourceTencentCloudGaapLayer4Listeners_UDP                                                         --- PASS: TestAccDataSourceTencentCloudGaapLayer4Listeners_UDP (31.57s)                                                === RUN   TestAccDataSourceTencentCloudGaapLayer7Listeners_basic                                                       --- PASS: TestAccDataSourceTencentCloudGaapLayer7Listeners_basic (26.12s)                                              === RUN   TestAccDataSourceTencentCloudGaapLayer7Listeners_https                                                       --- PASS: TestAccDataSourceTencentCloudGaapLayer7Listeners_https (18.99s)                                              === RUN   TestAccDataSourceTencentCloudGaapProxies_basic                                                               --- PASS: TestAccDataSourceTencentCloudGaapProxies_basic (144.59s)                                                     === RUN   TestAccDataSourceTencentCloudGaapProxies_filter                                                              --- PASS: TestAccDataSourceTencentCloudGaapProxies_filter (145.08s)                                                    === RUN   TestAccDataSourceTencentCloudGaapRealservers_basic                                                           --- PASS: TestAccDataSourceTencentCloudGaapRealservers_basic (1.39s)                                                   === RUN   TestAccDataSourceTencentCloudGaapRealservers_domain                                                          --- PASS: TestAccDataSourceTencentCloudGaapRealservers_domain (1.36s)                                                  === RUN   TestAccDataSourceTencentCloudGaapRealservers_name                                                            --- PASS: TestAccDataSourceTencentCloudGaapRealservers_name (1.39s)                                                    === RUN   TestAccDataSourceTencentCloudGaapSecurityPolices_basic                                                       --- PASS: TestAccDataSourceTencentCloudGaapSecurityPolices_basic (8.16s)                                               === RUN   TestAccDataSourceTencentCloudGaapSecurityRules_basic                                                         --- PASS: TestAccDataSourceTencentCloudGaapSecurityRules_basic (10.64s)                                                === RUN   TestAccDataSourceTencentCloudGaapSecurityRules_multi                                                         --- PASS: TestAccDataSourceTencentCloudGaapSecurityRules_multi (19.41s)                                                PASS                                                                                                                   ok      github.com/terraform-providers/terraform-provider-tencentcloud/tencentcloud     641.773s

## resource

> === RUN   TestAccTencentCloudGaapCertificate_basic                                                                     --- PASS: TestAccTencentCloudGaapCertificate_basic (1.12s)                                                             === RUN   TestAccTencentCloudGaapCertificate_clientCA                                                                  --- PASS: TestAccTencentCloudGaapCertificate_clientCA (0.68s)                                                          === RUN   TestAccTencentCloudGaapCertificate_ServerSSL                                                                 --- PASS: TestAccTencentCloudGaapCertificate_ServerSSL (0.66s)                                                         === RUN   TestAccTencentCloudGaapCertificate_realserverCA                                                              --- PASS: TestAccTencentCloudGaapCertificate_realserverCA (0.65s)                                                      === RUN   TestAccTencentCloudGaapCertificate_ProxySSL                                                                  --- PASS: TestAccTencentCloudGaapCertificate_ProxySSL (0.64s)                                                          === RUN   TestAccTencentCloudGaapCertificate_updateName                                                                --- PASS: TestAccTencentCloudGaapCertificate_updateName (0.96s)                                                        === RUN   TestAccTencentCloudGaapHttpDomain_basic                                                                      --- PASS: TestAccTencentCloudGaapHttpDomain_basic (26.39s)                                                             === RUN   TestAccTencentCloudGaapHttpDomain_https                                                                      --- PASS: TestAccTencentCloudGaapHttpDomain_https (27.19s)                                                             === RUN   TestAccTencentCloudGaapHttpDomain_httpsMutualAuthentication                                                  --- PASS: TestAccTencentCloudGaapHttpDomain_httpsMutualAuthentication (36.77s)                                         === RUN   TestAccTencentCloudGaapHttpDomain_httpsPolyClientCertificateIds                                              --- PASS: TestAccTencentCloudGaapHttpDomain_httpsPolyClientCertificateIds (36.61s)                                     === RUN   TestAccTencentCloudGaapHttpDomain_httpsCCIdToPolyIds                                                         --- PASS: TestAccTencentCloudGaapHttpDomain_httpsCCIdToPolyIds (28.06s)                                                === RUN   TestAccTencentCloudGaapHttpRule_basic                                                                        --- PASS: TestAccTencentCloudGaapHttpRule_basic (33.73s)                                                               === RUN   TestAccTencentCloudGaapHttpRule_httpUpdate                                                                   --- PASS: TestAccTencentCloudGaapHttpRule_httpUpdate (49.85s)                                                          === RUN   TestAccTencentCloudGaapHttpRule_httpUpdateRealservers                                                        --- PASS: TestAccTencentCloudGaapHttpRule_httpUpdateRealservers (40.11s)                                               === RUN   TestAccTencentCloudGaapHttpRule_noHealth                                                                     --- PASS: TestAccTencentCloudGaapHttpRule_noHealth (43.51s)                                                            === RUN   TestAccTencentCloudGaapHttpRule_domainRealserver                                                             --- PASS: TestAccTencentCloudGaapHttpRule_domainRealserver (38.21s)                                                    === RUN   TestAccTencentCloudGaapLayer4Listener_basic                                                                  --- PASS: TestAccTencentCloudGaapLayer4Listener_basic (30.12s)                                                         === RUN   TestAccTencentCloudGaapLayer4Listener_TcpDomain                                                              --- PASS: TestAccTencentCloudGaapLayer4Listener_TcpDomain (29.02s)                                                     === RUN   TestAccTencentCloudGaapLayer4Listener_update                                                                 --- PASS: TestAccTencentCloudGaapLayer4Listener_update (39.90s)                                                        === RUN   TestAccTencentCloudGaapLayer4Listener_udp                                                                    --- PASS: TestAccTencentCloudGaapLayer4Listener_udp (29.35s)                                                           === RUN   TestAccTencentCloudGaapLayer4Listener_udpDomain                                                              --- PASS: TestAccTencentCloudGaapLayer4Listener_udpDomain (29.01s)                                                     === RUN   TestAccTencentCloudGaapLayer4Listener_udpUpdate                                                              --- PASS: TestAccTencentCloudGaapLayer4Listener_udpUpdate (29.64s)                                                     === RUN   TestAccTencentCloudGaapLayer7Listener_basic                                                                  --- PASS: TestAccTencentCloudGaapLayer7Listener_basic (25.96s)                                                         === RUN   TestAccTencentCloudGaapLayer7Listener_https                                                                  --- PASS: TestAccTencentCloudGaapLayer7Listener_https (28.18s)                                                         === RUN   TestAccTencentCloudGaapLayer7Listener_httpsTwoWayAuthentication                                              --- PASS: TestAccTencentCloudGaapLayer7Listener_httpsTwoWayAuthentication (25.48s)                                     === RUN   TestAccTencentCloudGaapLayer7Listener_httpsForwardHttps                                                      --- PASS: TestAccTencentCloudGaapLayer7Listener_httpsForwardHttps (25.53s)                                             === RUN   TestAccTencentCloudGaapLayer7Listener_httpsPolyClientCertificateIds                                          --- PASS: TestAccTencentCloudGaapLayer7Listener_httpsPolyClientCertificateIds (27.90s)                                 === RUN   TestAccTencentCloudGaapLayer7Listener_httpsCcToPoly                                                          --- PASS: TestAccTencentCloudGaapLayer7Listener_httpsCcToPoly (27.19s)                                                 === RUN   TestAccTencentCloudGaapProxy_basic                                                                           --- PASS: TestAccTencentCloudGaapProxy_basic (153.86s)                                                                 === RUN   TestAccTencentCloudGaapProxy_update                                                                          --- PASS: TestAccTencentCloudGaapProxy_update (178.54s)                                                                === RUN   TestAccTencentCloudGaapRealserver_basic                                                                      --- PASS: TestAccTencentCloudGaapRealserver_basic (1.35s)                                                              === RUN   TestAccTencentCloudGaapRealserver_domain                                                                     --- PASS: TestAccTencentCloudGaapRealserver_domain (1.25s)                                                             === RUN   TestAccTencentCloudGaapRealserver_updateName                                                                 --- PASS: TestAccTencentCloudGaapRealserver_updateName (1.77s)                                                         === RUN   TestAccTencentCloudGaapRealserver_updateTags                                                                 --- PASS: TestAccTencentCloudGaapRealserver_updateTags (2.00s)                                                         === RUN   TestAccTencentCloudGaapSecurityPolicy_basic                                                                  --- PASS: TestAccTencentCloudGaapSecurityPolicy_basic (8.20s)                                                          === RUN   TestAccTencentCloudGaapSecurityPolicy_disable                                                                --- PASS: TestAccTencentCloudGaapSecurityPolicy_disable (8.46s)                                                        === RUN   TestAccTencentCloudGaapSecurityPolicy_drop                                                                   --- PASS: TestAccTencentCloudGaapSecurityPolicy_drop (7.79s)                                                           === RUN   TestAccTencentCloudGaapSecurityRule_basic                                                                    --- PASS: TestAccTencentCloudGaapSecurityRule_basic (11.01s)                                                           === RUN   TestAccTencentCloudGaapSecurityRule_drop                                                                     --- PASS: TestAccTencentCloudGaapSecurityRule_drop (10.95s)                                                            === RUN   TestAccTencentCloudGaapSecurityRule_name                                                                     --- PASS: TestAccTencentCloudGaapSecurityRule_name (12.85s)                                                            === RUN   TestAccTencentCloudGaapSecurityRule_ipSubnet                                                                 --- PASS: TestAccTencentCloudGaapSecurityRule_ipSubnet (11.02s)                                                        === RUN   TestAccTencentCloudGaapSecurityRule_allProtocols                                                             --- PASS: TestAccTencentCloudGaapSecurityRule_allProtocols (11.11s)                                                    === RUN   TestAccTencentCloudGaapSecurityRule_AllPorts                                                                 --- PASS: TestAccTencentCloudGaapSecurityRule_AllPorts (10.36s)                                                        PASS                                                                                                                   ok      github.com/terraform-providers/terraform-provider-tencentcloud/tencentcloud     1142.989s

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,23 @@
 FEATURES:
 * **New Resource**: `tencentcloud_kubernetes_as_scaling_group`.[#202](https://github.com/terraform-providers/terraform-provider-tencentcloud/issues/202)
 
+* Resource: `tencentcloud_gaap_layer4_listener` support import
+* Resource: `tencentcloud_gaap_http_rule` support import
+* Resource: `tencentcloud_gaap_security_rule` support import
```
Comment (oliverpei) [inline CHANGELOG.md]: 这些应该属于ENHANCEMENTS

**Pull request merged**